tests/test_cfg/test_z3_constraints.py

```diff
@@ -467,18 +467,25 @@ def func(x: int, y: int) -> None:
 
 def test_variable_reassignment() -> None:
```

Comment:
Rather than add to this test case, a better approach is to create a separate test case (to better isolate the new functionality being tested).